app/celery/tasks.py

```diff
@@ -73,18 +74,19 @@ def process_job(job_id):
     ).enumerated_recipients_and_personalisation:
         process_row(row_number, recipient, personalisation, template, job, service)
 
+    if template.template_type == LETTER_TYPE:
+        build_dvla_file.apply_async([str(job.id)], queue='process-job')
+        # temporary logging
+        current_app.logger.info("send job {} to build-dvla-file in the process-job queue".format(job_id))
+
     finished = datetime.utcnow()
-    job.job_status = JOB_STATUS_FINISHED
+    job.job_status = JOB_STATUS_FINISHED if template.template_type != LETTER_TYPE else job.job_status
```

Comment:
any reason you don't just put `job.job_status = JOB_STATUS_FINISHED` in an `else` two lines above? i think it'll make it look a little cleaner than a ternary conditional